test/InputOutput/unit_point.jl

```diff
@@ -0,0 +1,36 @@
+using Test
+import ClimaCore
+
```

Comment:
```suggestion
import ClimaCore: Spaces, Fields, InputOutput
```
Then use qualified modules below